src/components/CustomModal/index.jsx

```diff
@@ -21,7 +21,7 @@ const CustomModal = (props) => {
 	} = props;
 
 	return (
-		<ComposedModal open={showModal} onClose={() => closeModal(false)}>
+		<ComposedModal open={showModal} onClose={() => closeModal(false)} size="lg">
```

Comment:
Can we get size as a input prop since this looks like a common component

Reply:
thought of showing same pop-up size screen across screens